target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8074-rax120.dts

```diff
@@ -0,0 +1,1010 @@
+			reg = <0x280000 0x80000>;
+		};
+		partition@300000 {
+			label = "0:QSEE";
+			reg = <0x300000 0x300000>;
+		};
+		partition@600000 {
+			label = "0:QSEE_1";
+			reg = <0x600000 0x300000>;
+		};
+		partition@900000 {
+			label = "0:DEVCFG";
+			reg = <0x900000 0x80000>;
+		};
+		partition@980000 {
+			label = "0:DEVCFG_1";
+			reg = <0x980000 0x80000>;
+		};
+		partition@a00000 {
+			label = "0:APDP";
+			reg = <0xa00000 0x80000>;
+		};
+		partition@a80000 {
+			label = "0:APDP_1";
+			reg = <0xa80000 0x80000>;
+		};
+		partition@b00000 {
+			label = "0:RPM";
+			reg = <0xb00000 0x80000>;
+		};
+		partition@b80000 {
+			label = "0:RPM_1";
+			reg = <0xb80000 0x80000>;
+		};
+		partition@c00000 {
+			label = "0:CDT";
+			reg = <0xc00000 0x80000>;
+		};
+		partition@c80000 {
+			label = "0:CDT_1";
+			reg = <0xc80000 0x80000>;
+		};
+		partition@d00000 {
+			label = "0:APPSBLENV";
+			reg = <0xd00000 0x80000>;
+		};
+		partition@d80000 {
+			label = "0:APPSBL";
+			reg = <0xd80000 0x100000>;
+		};
+		partition@e80000 {
+			label = "0:APPSBL_1";
+			reg = <0xe80000 0x100000>;
+		};
+		partition@f80000 {
+			label = "0:ART";
+			reg = <0xf80000 0x80000>;
+		};
+		partition@100000 {
+			label = "0:ART.bak";
+			reg = <0x1000000 0x0080000>;
+		};
+		partition@1080000 {
+			label = "config";
+			reg = <0x1080000 0x0100000>;
+		};
+		partition@1180000 {
+			label = "boarddata1";
+			reg = <0x1180000 0x0100000>;
+		};
+		partition@1280000 {
+			label = "boarddata2";
+			reg = <0x1280000 0x0100000>;
+		};
+		partition@1380000 {
+			label = "pot";
+			reg = <0x1380000 0x0100000>;
+		};
+		partition@1480000 {
+			label = "dnidata";
+			reg = <0x1480000 0x0500000>;
+		};
+		partition@1980000 {
+			label = "firmware";
+			reg = <0x1980000 0x6400000>;
+		};
+		kernel@1980000 {
+			label = "kernel";
+			reg = <0x1980000 0x01D00000>;
+		};
+		rootfs@3680000 {
+			label = "rootfs";
+			reg = <0x3680000 0x4700000>;
+		};
+		partition@7d80000 {
+			label = "language";
+			reg = <0x7d80000 0x0380000>;
+		};
+		partition@8100000 {
+			label = "cert";
+			reg = <0x8100000 0x0100000>;
+		};
+		partition@8200000 {
+			label = "mtdoops";
+			reg = <0x8200000 0x0080000>;
+		};
+		partition@8280000 {
+			label = "router_analytics";
+			reg = <0x8280000 0x0200000>;
+		};
+		partition@8480000 {
+			label = "ntgrdata";
+			reg = <0x8480000 0x6400000>;
+		};
+		partition@E880000 {
+			label = "reserved";
+			reg = <0xE880000 0x11780000>;
+		};
+	};
+};
+
+&qusb_phy_0 {
+	status = "okay";
+};
+
+&qusb_phy_1 {
+	status = "okay";
+};
+
+&ssphy_0 {
+	status = "okay";
+};
+
+&ssphy_1 {
+	status = "okay";
+};
+
+&usb_0 {
+	status = "okay";
+	qcom,usb-dbm = <&dbm_1p5>;
+};
+
+&usb_1 {
+	status = "okay";
+};
+
+&wifi{
+	qcom,board_id = <0xff>;
+	qcom,ath11k-calibration-variant = "Netgear-RAX120";
+	status = "okay";
+};
+
+&cryptobam {
+	status = "okay";
+};
+
+&crypto {
+	status = "okay";
+};
+
+&prng {
+	status = "okay";
+};
+
+&edma {
+	status = "okay";
+};
+
+&tsens {
```

Comment:
This is already enabled by default in the SoC DTSI, drop the node

Reply:
Ahh yes it is defined in ipq8074 and enabled under the soc. Missed that. Thanks!! Correction pending.